roles/matrix-client-hydrogen/templates/config.json.j2

```diff
@@ -0,0 +1,3 @@
+{
+	"defaultHomeServer": {{ matrix_client_hydrogen_default_hs_url|string|to_json }}
```

Comment:
https://github.com/vector-im/hydrogen-web/blob/master/assets/config.json seems to use a domain name for `defaultHomeServer`, not a full URL like we currently are. Maybe both work? Maybe not.

Reply:
I've tested. Both work.